source/testing/testing-components.md

```diff
@@ -262,11 +312,11 @@ To stub the location service in your test, create a local stub object that exten
 `Ember.Service`, and register the stub as the service your tests need in the
```

Comment:
@Turbo87  Here's also the updated example on how to stub Services - can you take a look if this is good to go?